### ISSUE TITLE
Saved filter sets: Autofocus makes workflow with keyboard and screen reader difficult

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#1219]
If you select Add, Cancel is then automatically selected. The screen reader correctly reads out “Cancel”, but you would actually want to make an entry. So you have to process the form backwards with shift + tab. As there is no focus trap, it is not possible to return to the beginning of the action menu with tab.

**Comment by Arnei** [comment]
Fixed by #1336
